app/services/invite_schools.rb

```diff
@@ -251,6 +242,29 @@ def invite_cip_only_schools
     end
   end
 
+  def invite_sitless_opted_out_schools_for_nqt_plus_one
+    School.eligible.opted_out.where.missing(:induction_coordinators).each do |school|
+      if school.contact_email.blank?
+        logger.info "No contact details for school urn: #{school.urn} ... skipping"
+        next
+      end
+
+      SchoolMailer.nqt_plus_one_sitless_invite(
+        recipient: school.contact_email,
+        start_url: year2020_start_url(school, utm_source: :year2020_nqt_invite_school),
+      ).deliver_later
+    end
+  end
+
+  def invite_opted_out_sits_for_nqt_plus_one
+    School.eligible.opted_out.joins(:induction_coordinators).each do |school|
+      SchoolMailer.nqt_plus_one_sit_invite(
+        recipient: school.induction_coordinators.first.email,
+        start_url: year2020_start_url(school, utm_source: :year2020_nqt_invite_sit),
+      ).deliver_later
+    end
+  end
```

Comment:
For both of those, should we check whether the schools added 2020 participants already? And if they have, not send the email?

Reply:
Discussed on slack, nobody used it yet so we can send it to everyone who opted out.